### PR TITLE
Form updates

### PR DESCRIPTION
**Pull request by JessLayton**

### CODE REVIEW

**Review by lukecottenham — APPROVED** [review]

File: hack-hate-inclusion-london/src/components/ddpo/OrgEntry.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const OrgEntry = () => {
     const { valid, message } = validateEntry(orgName);
     if (valid) {
       const response = await addOrganisation(orgName);
-      if (response && response.data && response.data.status === 'success') {
+      if (response && response.data && response.data.data) {
```
Comment (Harr1s96) [inline hack-hate-inclusion-london/src/components/ddpo/OrgEntry.js:42]: Do we really need to check for response and response.data objects? Can we not just check for response.data.data?

**Review by Harr1s96 — APPROVED** [review]
Other than the comment I made  on the 'if statement' in _OrgEntry.js_  This looks pretty good.

File: hack-hate-inclusion-london/src/components/ddpo/OrgEntry.js (hunk 1 above)
Comment (lukecottenham) [reply hack-hate-inclusion-london/src/components/ddpo/OrgEntry.js:42]: If we don't and the response body comes back empty then we'll get uninformative errors with cannot access property data of undefined

Comment (lukecottenham) [reply hack-hate-inclusion-london/src/components/ddpo/OrgEntry.js:42]: Though if we update to ES2020 there is a more effective way to do this

**Pull request merged**